weave/integrations/openai_agents/openai_agents.py

```diff
@@ -71,6 +73,9 @@ def __init__(self) -> None:
 
     def on_trace_start(self, trace: tracing.Trace) -> None:
         """Called when a trace starts."""
+        if not (wc := get_weave_client()):
```

Comment:
```suggestion
        if (weave_client := get_weave_client()) is None:
```
More explicit naming and explicit `None` check. Same below.